trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    if suer.length > 10
+      super[0...10]
+    else
+      super
+    end
+  end
```

Comment:
- [ ] Kindly fix this typo error in line 5 as it's called `super` instead of `suer`.
![image](https://user-images.githubusercontent.com/24703171/225138755-c792071c-94f6-4a26-bf04-ec22b13aecd9.png)